lib/questions.rb

```diff
@@ -63,33 +84,41 @@ def separate_array_into_even_and_odd_numbers(array)
 # e.g. 'bob'. So in the array ['bob', 'radar', 'eat'], there
 # are 2 palindromes (bob and radar), so the method should return 2
 def number_of_elements_that_are_palindromes(array)
+  (array.select{|word| word == word.reverse}).count
 end
 
 # return the shortest word in an array
 def shortest_word_in_array(array)
+  array.min_by{|word| word.length}
 end
 
 # return the shortest word in an array
 def longest_word_in_array(array)
+    array.max_by{|word| word.length}
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:length` as an argument to `max_by` instead of a block.